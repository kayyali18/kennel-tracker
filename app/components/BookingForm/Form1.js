import styled from 'styled-components'

const Form1 = styled.form`
  background-color: beige;
  display: grid;
  grid-gap: 4rem;
  grid-template-areas: 'input .';
  height: 100%;

  position: absolute;
  overflow: scroll;
  width: 100%;
  input {
    border: 2px solid black;
    position: relative;
    height: 50px;
  }

  input:nth-child(odd) {
    top: 2rem;
    left: 4rem;
  }
  input:nth-child(even) {
    top: 6rem;
    right: 4rem;
  }
`
export default Form1

// make them checkboxes so when checked they can move to the next
