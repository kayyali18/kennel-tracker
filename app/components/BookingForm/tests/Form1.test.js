import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Form1 from '../Form1'

describe('<Form1 />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderer.create(<Form1 />).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })

  it('should render a form', () => {
    const renderedComponent = renderer.create(<Form1 />).toJSON()
    expect(renderedComponent.type).toEqual('form')
  })
})
