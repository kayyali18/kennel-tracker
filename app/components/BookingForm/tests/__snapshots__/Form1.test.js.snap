// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form1 /> should match snapshot 1`] = `
.c0 {
  display: grid;
  grid-gap: 4rem;
  grid-template-areas: 'input .';
  height: 100%;
  position: absolute;
  overflow: scroll;
  width: 100%;
}

.c0 input {
  border: 2px solid black;
  position: relative;
  height: 50px;
}

.c0 input:nth-child(odd) {
  top: 2rem;
  left: 4rem;
}

.c0 input:nth-child(even) {
  top: 6rem;
  right: 4rem;
}

<form
  className="c0"
/>
`;
